src/main.rs

```diff
@@ -12,8 +12,12 @@ use structopt::StructOpt;
 )]
 struct CliOpts {
     /// The address that the mock destination service will listen on.
-    #[structopt(short = "a", long = "addr", default_value = "0.0.0.0:8086")]
-    addr: SocketAddr,
+    #[structopt(long = "dst_addr", default_value = "0.0.0.0:8086")]
+    dst_addr: SocketAddr,
+
+    /// The address that the mock identity service will listen on.
+    #[structopt(long = "identity_addr", default_value = "0.0.0.0:8080")]
+    identity_addr: SocketAddr,
```

Comment:
Do we actually need different server ports? Can we just expose a single gRPC server?

Reply:
If we need separate ports, would prefer `dst-addr` to `dst_addr` -- underscores a little odd in CLI flags

Reply:
Yea I could change it so that `DstService` impls both `Destination` and `Identity`. There isn't a particular reason I separated them, but it should be fairly simple to change it so that only one server needs to run.